lokimq/lokimq.h

```diff
@@ -793,6 +816,31 @@ class LokiMQ {
      */
     void add_command_alias(std::string from, std::string to);
 
+    /** Creates a "tagged thread" and starts it immediately.  A tagged thread is one that batches,
+     * jobs, and timer jobs can be sent to by specifically, typically to perform coordination of
```

Comment:
"by specifically, " -- missing a word here?

Reply:
Too *many* words.  Dropped the "by".